docker-compose.yml

```diff
@@ -0,0 +1,55 @@
+version: "3.7"
+services:
+  nginx:
+    image: nginx:1.19-alpine
+    ports:
+      - "80:80"
+    volumes:
+      - ./configurations/30-sl-nginx.entrypoint.sh:/docker-entrypoint.d/30-sl-nginx.entrypoint.sh:ro
+  postgres: # Must be identical to the .env POSTGRES_HOST variable
+    image: postgres:12.1
+    environment:
+      POSTGRES_USER: ${POSTGRES_DB:?Required}
+      POSTGRES_PASSWORD: ${POSTGRES_PASSWORD:?Required}
+      POSTGRES_DB: ${POSTGRES_DB:?Required}
+      DB_HOST: ${POSTGRES_HOST:?Required}
+  postfix: # temporary
+    image: arugifa/simplelogin-postfix:3.1.0-6
```

Comment:
maybe can fork this repo to SimpleLogin GitHub account and build an image from there (something like `simplelogin/postfix:1.0.0`), what do you think?

Reply:
can this image be used without lets encrypt ?
i have Nginx proxy manager handling certificates